packages/react-component-library/src/tokens/typography.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta, Source } from '@storybook/blocks'
+import { TypographyTable } from './TypographyTable'
+
+
+
+<Meta title="Tokens/Typography" />
+
+# Typography
+
+All Royal Navy Design System applications are set in [Lato](https://www.latofonts.com/).
```

Comment:
This is a dead link to a broken website - can we remove please.

Reply:
The link works for me?

Reply:
Must have just been down when I hit it.